coriolis_openstack_utils/cli/subnets.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2018 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from oslo_log import log as logging
+
+from cliff import lister
+
+from coriolis_openstack_utils import actions
+from coriolis_openstack_utils import conf
+from coriolis_openstack_utils.cli import formatter
+from coriolis_openstack_utils import networks
+from coriolis_openstack_utils import subnets
+
+
+LOG = logging.getLogger(__name__)
+
+
+class SubnetMigrationFormatter(formatter.EntityFormatter):
+    columns = (
+        "Subnet Name",
+        "Subnet ID",
+        "Network ID",
+        "Network Name")
+
+    def _get_formatted_data(self, obj):
+        data = (
+            obj["destination_name"],
+            obj["destination_id"],
+            obj["dest_network_id"],
+            obj["dest_network_name"])
+
+        return data
+
+
+class MigrateSubnet(lister.Lister):
+    def get_parser(self, prog_name):
+        parser = super(MigrateSubnet, self).get_parser(prog_name)
+        parser.add_argument(
+            "--src-network-id", dest="src_network_id",
+            default=False,
+            help="The source network id of the source subnet.")
+        parser.add_argument(
+            "--dest-network-id", dest="dest_network_id",
+            default=False,
+            help="The destination network id where the subnet is located")
+        parser.add_argument(
+            "--not-a-drill", dest="not_drill", action="store_true",
+            default=False,
+            help="If unset, tooling will only print the indented operations.")
+        parser.add_argument("subnet_name", metavar="SUBNET_NAME")
+
+        return parser
+
+    def take_action(self, args):
+        # instantiate all clients:
+        source_client = conf.get_source_openstack_client()
+        destination_client = conf.get_destination_openstack_client()
+
+        subnet_creation_payload = {
+            "source_name": args.subnet_name,
+            "src_network_id": args.src_network_id,
```

Comment:
Please allow the user to specify the source network by name as well (just do a `neutron.find_resource()`)